packages/event-migration/src/index.ts

```diff
@@ -271,12 +271,18 @@ const { parseEventType, decodeEvent, parseId } = match(config.targetDbSchema)
   )
   .exhaustive();
 
+let skippedEvents = 0;
 for (const event of originalEvents) {
   console.log(event);
   const { event_ser_manifest, event_payload, write_timestamp } = event;
 
   const parsedEventType = parseEventType(event_ser_manifest);
 
+  if (parsedEventType === "") {
```

Comment:
may I ask to add a comment for this case?

Reply:
Done